packages/round-manager/src/features/program/ViewProgramPage.tsx

```diff
@@ -360,20 +354,19 @@ export default function ViewProgram() {
                 </Button>
               </Link>
             )}
-            {grantType === "directGrant" &&
-              process.env.REACT_APP_DIRECT_GRANT_ENABLED && (
-                <Link
-                  to={`/round/create?programId=${programToRender?.id}&roundCategory=direct`}
-                >
-                  <Button className="px-4 w-full h-[48px]">
-                    <PlusIcon
-                      className="h-4 w-4 inline-flex -translate-y-0.5"
-                      aria-hidden="true"
-                    />
-                    &nbsp;Create round
-                  </Button>
-                </Link>
-              )}
+            {grantType === "directGrant" && (
+              <Link
+                to={`/round/create?programId=${programToRender?.id}&roundCategory=direct`}
```

Comment:
If `programToRender` is undefined, will the link be valid?

Thinking aloud / not in scope here: would be nice to have routes checked at compile time with something like:

```tsx
<Link to={routes.round.create({ programId: programToRender.id, roundCategory: 'direct' })
```

Reply:
Static route checking is on the roadmap for sure. I'll see if we can do more check around programtorender here.

Reply:
It's very convoluted and not aparent, but this part of the UI only renders when programToRender is not null, via a series of state-that-should-be-a-variable checks. So this is a non-issue, but it for sure needs to be cleaned up and checked at compile-time, not review-time.